crates/astria-bridge-withdrawer/ethereum/local.env.example

```diff
@@ -0,0 +1,20 @@
+# anvil account 0
+PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

Comment:
How are these env vars used with contract deployment? Is this foundry env var? Generally we want to put private keys in files.

Reply:
I agree. We should never leak private keys in plain text.

Reply:
this is anvil account 0 so it's a known private key. i can remove tho